paint-timing/fcp-only/fcp-canvas-context.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<head>
+<title>Performance Paint Timing Test: FCP when canvas context is created</title>
+</head>
+<body>
+<style>#log {display: none}</style>
+<script src="../resources/utils.js"></script>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<canvas id="canvas" width="50" height="50"></canvas>
+</body>
+
+<footer>
+<script>
+  promise_test(async t => {
+    try {
+        assert_precondition(window.PerformancePaintTiming, "Paint Timing isn't supported.");
+        await new Promise(r => window.addEventListener('load', r))
+        await assertNoFirstContentfulPaint(t)
+        const canvas = document.getElementById('canvas');
+        const context = canvas.getContext('2d')
+        context.fillRect(0, 0, 100, 100)
+        await assertFirstContentfulPaint(t)
+    } catch (e) {
```

Comment:
Why is this try/catch/finally needed? After `await assertFirstContentfulPaint` it does not matter if we log text to the test. Probably remove?

Reply:
it's needed so that we show the log DIV if the test fails, it really helps with debugging